app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor.get('name', 'Visitor')}"
+                                     f" is not vaccinated.")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
```

Comment:
The comparison `if expiration_date < datetime.date.today():` assumes that `expiration_date` is always a valid date object. Ensure that `expiration_date` is not `None` before performing this comparison to avoid potential `TypeError`. Consider adding a check to handle cases where `expiration_date` might be missing or invalid.